src/Controls/tests/TestCases.Shared.Tests/Tests/Issues/XFIssue/Issue3809.cs

```diff
@@ -1,67 +1,69 @@
 ﻿using NUnit.Framework;
+using NUnit.Framework.Legacy;
 using UITest.Appium;
 using UITest.Core;
 
 namespace Microsoft.Maui.TestCases.Tests.Issues
 {
 	public class Issue3809 : _IssuesUITest
 	{
+		const string SetPagePadding = "Set Page Padding";
+		const string SafeAreaText = "Safe Area Enabled: ";
+		const string PaddingLabel = "paddingLabel";
+		const string SafeAreaAutomationId = "SafeAreaAutomation";
+
 		public Issue3809(TestDevice testDevice) : base(testDevice)
 		{
 		}
 
 		public override string Issue => "SetUseSafeArea is wiping out Page Padding ";
 
-		// TODO: The _ variables need to be AutomationId values
-		//void AssertSafeAreaText(string text)
-		//{
-		//	var element =
-		//		RunningApp
-		//			.WaitForFirstElement(_safeAreaAutomationId);
-
-		//	element.AssertHasText(text);
-		//}
-
-		//[Test]
-		//[Category(UITestCategories.Layout)]
-		//public void SafeAreaInsetsBreaksAndroidPadding()
-		//{
-		//	// ensure initial paddings are honored
-		//	AssertSafeAreaText($"{_safeAreaText}{true}");
-		//	var element = App.WaitForFirstElement(_paddingLabel);
+		void AssertSafeAreaText(string text)
+		{
+			var element = App.WaitForFirstElement(SafeAreaAutomationId);
 
-		//	bool usesSafeAreaInsets = false;
-		//	if (element.ReadText() != "25, 25, 25, 25")
-		//		usesSafeAreaInsets = true;
+			ClassicAssert.AreEqual(element.GetText(), text);
+		}
 
-		//	Assert.AreNotEqual(element.ReadText(), "0, 0, 0, 0");
-		//	if (!usesSafeAreaInsets)
-		//		Assert.AreEqual(element.ReadText(), "25, 25, 25, 25");
+		[Test]
+		[Category(UITestCategories.Layout)]
+		[Category(UITestCategories.Page)]
+		public void SafeAreaInsetsBreaksAndroidPadding()
+		{
+			// Ensure initial paddings are honored
+			AssertSafeAreaText($"{SafeAreaText}{true}");
+			var element = App.WaitForFirstElement(PaddingLabel);
 
-		//	// disable Safe Area Insets
-		//	App.Tap(_safeAreaAutomationId);
-		//	AssertSafeAreaText($"{_safeAreaText}{false}");
-		//	element = App.WaitForFirstElement(_paddingLabel);
+			bool usesSafeAreaInsets = false;
+			if (element.ReadText() != "25, 25, 25, 25")
+				usesSafeAreaInsets = true;
 
-		//	Assert.AreEqual(element.ReadText(), "25, 25, 25, 25");
+			ClassicAssert.AreNotEqual(element.ReadText(), "0, 0, 0, 0");
+			if (!usesSafeAreaInsets)
+				ClassicAssert.AreEqual(element.ReadText(), "25, 25, 25, 25");
 
-		//	// enable Safe Area insets
-		//	App.Tap(_safeAreaAutomationId);
-		//	AssertSafeAreaText($"{_safeAreaText}{true}");
-		//	element = App.WaitForFirstElement(_paddingLabel);
-		//	Assert.AreNotEqual(element.ReadText(), "0, 0, 0, 0");
+			// Disable Safe Area Insets
+			App.Tap(SafeAreaAutomationId);
```

Comment:
**Step 4**: Find replacement methods in Appium. See the reference table from the PR. We already have a lot of helper methods to interact with the UI through Appium. You can find those under src\TestUtils\src\UITest.Appium.

If you do think there is no alternative, please reach out and lets see if we need to add something.